crates/trie/sparse/src/state.rs

```diff
@@ -1,25 +1,24 @@
 use crate::{
     blinded::{BlindedProvider, BlindedProviderFactory, DefaultBlindedProviderFactory},
     metrics::SparseStateTrieMetrics,
-    RevealedSparseTrie, SparseTrie, TrieMasks,
+    trace, RevealedSparseTrie, SparseTrie, TrieMasks,
 };
 use alloy_primitives::{
     hex,
     map::{B256Map, HashMap, HashSet},
     Bytes, B256,
 };
 use alloy_rlp::{Decodable, Encodable};
-use core::fmt;
+use core::{fmt, iter::Peekable};
 use reth_execution_errors::{SparseStateTrieErrorKind, SparseStateTrieResult, SparseTrieErrorKind};
 use reth_primitives_traits::Account;
-use reth_tracing::tracing::trace;
 use reth_trie_common::{
     proof::ProofNodes,
     updates::{StorageTrieUpdates, TrieUpdates},
     MultiProof, Nibbles, RlpNode, StorageMultiProof, TrieAccount, TrieMask, TrieNode,
     EMPTY_ROOT_HASH, TRIE_ACCOUNT_RLP_MAX_SIZE,
 };
-use std::{collections::VecDeque, iter::Peekable};
+use std::{collections::VecDeque, vec::Vec};
```

Comment:
unsure if vecdeque is no-std compat

Reply:
I think its like Vec, so its no_std + alloc

Reply:
I tested using ` cargo check --target riscv32imac-unknown-none-elf -p reth-trie-sparse --no-default-features`